Basic/DyanmicPrefabs/Assets/Scripts/02_Server Authoritative Preload All Prefabs Asynchronously/LoadAllPrefabs.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+
+namespace Game.LoadAllPrefabs
+{
+    /// <summary>
+    /// A simple use case where the server notifies all clients to preload a collection of network prefabs. The server
+    /// will not invoke a spawn in this use case, and will incrementally load each dynamic prefab, one prefab at a time.
```

Comment:
Why no spawn? I'd expect to load all, then once we get the confirmation that all clients have loaded, to then spawn the object, no?

Reply:
Oh wait, that's the case where we just do some preloading before doing some spawns later, correct? Might be worth it to describe the full expected flow in the comments

Reply:
That's what the previous implementation was. The button served to only instruct users to load a batch of network prefabs all at once, without spawn. Should it?

Reply:
just to confirm, the flow would be "clients and host are all connected. The host arrives at a point in the game where they know some prefabs will be needed soon. They tell all clients to load those prefabs. A few minutes later, the server can do a simple spawn and all is setup already, nothing to worry about. This allows for simple spawn management." ?
We either need to describe this in doc or give an example in code. Right now this isn't clear.

Reply:
Yes I see. That can be captured with some remarks on the "why". I'll add that.